src/atoms/Photography/photography.module.css

```diff
@@ -0,0 +1,8 @@
+.photography {
+  text-align: center;
+  background-image: url("../../images/image_1.jpg");
```

Comment:
Rename this image to something more descriptive.

Reply:
Renamed to students.jpg